tests/test_utils_strptime.py

```diff
@@ -72,12 +72,18 @@ def test_parsing_date_should_fail_using_datetime_strptime_if_locale_is_non_engli
         param('12 Dec 10 10:30:55.10', '%d %b %y %H:%M:%S.%f', expected=datetime(2010, 12, 12, 10, 30, 55, 100000)),
         param('12 Dec 10 10:30:55.100', '%d %b %y %H:%M:%S.%f', expected=datetime(2010, 12, 12, 10, 30, 55, 100000)),
         param('12 Dec 10 10:30:55.1000', '%d %b %y %H:%M:%S.%f', expected=datetime(2010, 12, 12, 10, 30, 55, 100000)),
-        param('12 Dec 10 10:30:55.100000', '%d %b %y %H:%M:%S.%f', expected=datetime(2010, 12, 12, 10, 30, 55, 100000)),
+        param(
+            '12 Dec 10 10:30:55.100000', '%d %b %y %H:%M:%S.%f',
+            expected=datetime(2010, 12, 12, 10, 30, 55, 100000)
+        ),
         param('12 Dec 10 10:30:55.000001', '%d %b %y %H:%M:%S.%f', expected=datetime(2010, 12, 12, 10, 30, 55, 1)),
         param('12 Dec 10 10:30:55.000011', '%d %b %y %H:%M:%S.%f', expected=datetime(2010, 12, 12, 10, 30, 55, 11)),
         param('12 Dec 10 10:30:55.000111', '%d %b %y %H:%M:%S.%f', expected=datetime(2010, 12, 12, 10, 30, 55, 111)),
         param('12 Feb 2016 11:41:23', '%d %b %Y %I:%M:%S', expected=datetime(2016, 2, 12, 11, 41, 23)),
-        param('11 Dec 10 10:30:2011.999999', '%y %b %S %H:%M:%Y.%f', expected=datetime(2011, 12, 1, 10, 30, 10, 999999)),
+        param(
+            '11 Dec 10 10:30:2011.999999', '%y %b %S %H:%M:%Y.%f',
+            expected=datetime(2011, 12, 1, 10, 30, 10, 999999)
+        ),
```

Comment:
what about make the same change in the other lines?

Reply:
@eamanu because it would add a lot of noise to this PR, making it hard to read.